obspy/core/inventory/response.py

```diff
@@ -166,9 +166,6 @@ def __str__(self):
                 if self.decimation_input_sample_rate is not None else ""))
         return ret.strip()
 
-    def _repr_pretty_(self, p, cycle):
```

Comment:
This doesn't relate to AttribDict, just another place that can use some polishing. I've moved `_repr_pretty_` to the base class `ComparingObject` and also enhanced it some.